bookstore/publish.py

```diff
@@ -33,20 +37,42 @@ def initialize(self):
             raise web.HTTPError(400, "Must provide a path for publishing")
 
         model = self.get_json_body()
-        if model:
-            await self._publish(model, path.lstrip('/'))
-        else:
-            raise web.HTTPError(400, "Cannot publish an empty model")
+        self.validate_model(model)
+        resp = await self._publish(model['content'], path.lstrip('/'))
+        self.set_status(201)
```

Comment:
Is there a particular reason that we are setting the status as 201 instead of taking from the response?

Reply:
Do you mean setting the status to match the response from the S3 client? We can do that… we hadn't been doing that for the error handling (because from our server's perspective we didn't have the same kind of error)… but I suppose the successful case can be passed through.

But for now I'll need to dive into the code to see what the actual response is.